packages/extension/src/components/atoms/NumericInput/NumericInput.tsx

```diff
@@ -1,11 +1,19 @@
-import { ChangeEvent, FC, useCallback, useState } from 'react';
+import { ChangeEvent, FC, useCallback, useState, useEffect } from 'react';
```

Comment:
```suggestion
import { ChangeEvent, FC, useCallback, useState } from 'react';
```
Following the comment bellow.